### PR TITLE
Feature/query log control

### PR DESCRIPTION
**Pull request by kobemertens**
Overrides https://github.com/mu-semtech/mu-python-template/pull/17

Allow configuration of read and update queries separately

### CODE REVIEW

File: helpers.py [hunk 1]
```diff
@@ -42,6 +42,16 @@
 consoleHandler = logging.StreamHandler(stream=sys.stdout)# or stderr?
 logger.addHandler(consoleHandler)
 
+LOG_SPARQL_ALL_VAR = os.environ.get('LOG_SPARQL_ALL') 
+LOG_SPARQL_QUERIES = os.environ.get(
+    'LOG_SPARQL_QUERIES',
+    default=LOG_SPARQL_ALL_VAR
+).lower() == 'true'
```
Comment (MikiDi) [inline helpers.py:49]: Since the Python boolean syntax is `True` (capitalized) I suggest adding that one too.

Comment (MikiDi) [reply helpers.py:49]: nvm. I didn't notice the lowercasing method.

**Review by MikiDi — APPROVED** [review]

**Comment by kobemertens** [comment]
@MikiDi Does this need more review or can we merge this? :)

**Comment by MikiDi** [comment]
I indicated my approval for merging this PR as-is by submitting an approval review. I don't have the rights to merge myself however.

**Pull request merged**